.travis.yml

```diff
@@ -11,11 +11,14 @@ services:
 before_install:
   - sudo apt-get update 
   - sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install docker-ce
-  - sudo apt-get install golang -y
+  - sudo apt-get install golang wget -y
   - docker version
+  - wget -qO - https://packages.confluent.io/deb/5.2/archive.key | sudo apt-key add -
```

Comment:
I think curl is good enough to download, do have to download wget

Reply:
curl may not be installed in the travis vm. I use the official command in confluent to set that key.
what is the version of ubuntu used in travis? if we are sure that curl is already installed in travis vm, i will change to use curl instead of wget. Thanks.

Reply:
@thesues i have tried just now, and curl is already installed in the vm used by travis. Please help to review this PR again, thx.